### DISCUSSION TITLE
colab popped up a warning for me today, anyone else?

### DISCUSSION THREAD

**Discussion by Headupdef** [TheLastBen/fast-stable-diffusion#2014] [Q&A]
does this mean I am flagged or is this something new?

the popup said this:

Warning
You may be executing code that is disallowed, and this may restrict your ability to use Colab in the future. Please note the prohibited actions specified in our FAQ.

FAQ says:

Resources in Colab are prioritized for interactive use cases. We prohibit actions associated with bulk compute, actions that negatively impact others, as well as actions associated with bypassing our policies. The following are disallowed from Colab runtimes:

file hosting, media serving, or other web service offerings not related to interactive compute with Colab
downloading torrents or engaging in peer-to-peer file-sharing
remote control such as SSH shells, remote desktops, remote UIs
connecting to remote proxies
mining cryptocurrency
running denial-of-service attacks
password cracking
using multiple accounts to work around access or resource usage restrictions
creating deepfakes

**Comment by HaikenEdge** [comment]
[Look here.](https://github.com/TheLastBen/fast-stable-diffusion/issues/2013)

**Comment by TheLastBen** [comment]
use the latest notebook

**Reply by tirtho888** [reply]
Yeah, no warning for me

**Reply by i-huzaifa-arshad** [reply]
I currently am running previous notebook (the one before the latest) and I did got the warning but i ignored and continued anyway. It is working fine for me.

**Reply by tirtho888** [reply]
You should use the new notebook, or you might get banned from Google Colab.

**Reply by TheLastBen** [reply]
no one is getting banned, at least for the time being

**Reply by jorymo** [reply]
I'm still getting the warning on the new one

**Reply by TheLastBen** [reply]
which cell ? use this official link https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb

**Comment by alperinugur** [comment]
I got the same message. Should we check the  "Use_Latest_Working_Commit" checkbox? I did not check that one.

**Reply by TheLastBen** [reply]
use the latest notebook by clicking on the thumbnail on the main page of the repo
